code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -402,4 +402,4 @@ datum/chemical_reaction/flash_powder
 	id = "thermite"
 	result = "thermite"
 	required_reagents = list("aluminum" = 1, "iron" = 1, "oxygen" = 1)
-	result_amount = 3
\ No newline at end of file
+	result_amount = 3
```

Comment:
Newline stuff. The automerger lumped the reagent change over to the recipes folder, instead of the reagents folder, so I had to toss it over.